src/test/llmtemplate_test.cpp

```diff
@@ -38,6 +42,28 @@
 
 using namespace ovms;
 
+class MockedServerRequestInterface final : public tensorflow::serving::net_http::ServerRequestInterface {
```

Comment:
Already exists in two places - llmnode_test and http_openai_handler_test, and now we introduce into third test case, maybe we could separate to test utils?